opentelemetry-appender-log/src/lib.rs

```diff
@@ -61,7 +60,7 @@ where
     }
 }
 
-fn map_severity_to_otel_severity(level: Level) -> Severity {
+const fn severity_of_level(level: Level) -> Severity {
```

Comment:
nit - Can this take reference to `Level` ?

Reply:
It can, it's just that its a `Copy` item and the `log` crate returns them by value, so ergonomically it was easier to not pass by reference.

Yes, the other file passes it by reference, but that is because the `tracing` API to get the level returns a reference to Level.